.travis.yml

```diff
@@ -1,16 +1,10 @@
-before_install:
-  - nvm install-latest-npm
```

Comment:
this should be kept to ensure all relevant npm bugfixes are available.

Reply:
Is travis not going to use the latest npm? That's unfortunate.

Reply:
no CI systems ever do by default; the npm version is only the one that comes with the CI job's node version, unless you manually update it; however, `npm install -g npm` isn't reliable, which is why `nvm install-latest-npm` exists.